multiqc/modules/featureCounts/feature_counts.py

```diff
@@ -29,10 +29,11 @@ def __init__(self):
         for f in self.find_log_files(config.sp['featurecounts']):
             parsed_data = self.parse_featurecounts_report(f['f'])
             if parsed_data is not None:
-                if f['s_name'] in self.featurecounts_data:
+                s_name = f['s_name'].replace(".counts.summary", "")
```

Comment:
Might be nicer to add `.counts` to `fn_clean_exts` in `config.py` rather than hardcoding here? (I'm probably being a hypocrite, I'm sure I've done the same in other modules)